test/db/analysis/avr

```diff
@@ -809,7 +809,6 @@ EXPECT=<<EOF
 0x0000246e 34 fcn.0000246e
 0x000025dc 156 fcn.000025dc
 0x0000268c 138 fcn.0000268c
-0x00002818 78 fcn.00002818
```

Comment:
Where does this come from?

Reply:
this was the result of esil computation and the address is pointing to a subroutine, but the problem is that is pointing in the middle of it, which is wrong. the issue is that the esil code tried to emulate some instructions where loops matters and not executing the whole loop resulted in this wrong output. you can easily verify this by seeking to that function and looking backwards for the first sequence of ret ; pop ;pop ;pop which is used for interrupt handling.